src/components/pages/Photos/cameraNavbar/CameraNavbar.css

```diff
@@ -0,0 +1,20 @@
+.cameranavbar {
```

Comment:
These class names should be separated with hyphens, e.g. `camera-navbar` and `camera-box` 😊